releasenotes/notes/deprecate-span-meta-ad1c96ba55acaac9.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    :py:attr:`ddtrace.Span.meta` has been deprecated. Use :py:meth:`ddtrace.Span.get_tag` and :py:meth:`ddtrace.Span.set_tag` instead.
```

Comment:
Can we confirm that these render ok in the generated release notes?

Reply:
This is how they're generated in the release notes: https://github.com/DataDog/dd-trace-py/releases/edit/untagged-61093626ab9d299f9f0b. Is this expected?

Reply:
I mean the generated release notes that'll appear here: https://ddtrace.readthedocs.io/en/stable/release_notes.html